packages/fxa-content-server/app/scripts/templates/partial/show-password.mustache

```diff
@@ -1,2 +1,4 @@
 <input id="{{ id }}" type="checkbox" class="show-password" aria-controls="{{ targetId }}" tabindex="-1" data-novalue data-synchronize-show="{{ synchronizeShow }}"/>
-<label for="{{ id }}" class="show-password-label" title="{{#t}}Show password{{/t}}"><img alt="{{#t}}Show password{{/t}}" src="data:image/svg+xml;base64,PHN2ZyBkYXRhLW5hbWU9IkxheWVyIDIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjU1IiBoZWlnaHQ9IjQ2IiB2aWV3Qm94PSItMTYgLTEzIDQ3IDM4Ij48cGF0aCBkPSJNMTYgNS43QTguMzIgOC4zMiAwIDAgMCA4IDAgOC4zMiA4LjMyIDAgMCAwIC4wNSA1LjdhLjg2Ljg2IDAgMCAwIDAgLjU5QTguMzEgOC4zMSAwIDAgMCA4IDEyYTguMzIgOC4zMiAwIDAgMCA4LTUuNy44OC44OCAwIDAgMCAwLS42ek04LjUgM0ExLjUgMS41IDAgMSAxIDcgNC41IDEuNSAxLjUgMCAwIDEgOC41IDN6TTggMTBhNi4zMyA2LjMzIDAgMCAxLTUuOTQtNCA2LjA4IDYuMDggMCAwIDEgMi41LTNBNCA0IDAgMCAwIDQgNWE0IDQgMCAwIDAgOCAwIDQgNCAwIDAgMC0uNTYtMiA2IDYgMCAwIDEgMi41IDNBNi4zMyA2LjMzIDAgMCAxIDggMTB6Ii8+PC9zdmc+Cg=="></label>
+<label for="{{ id }}" class="show-password-label" title="{{#t}}Show password{{/t}}">
```

Comment:
This probably needs an `aria-label` saying "Show password", I don't think it'll read just "title".